modules/virt-analyzing-datavolume-conditions-and-events.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-analyzing-datavolumes-using-events-and-conditions.adoc
+
+[id="virt-analyzing-datavolume-conditions-and-events_{context}"]
+= Analyzing DataVolumes using conditions and events
+
+By inspecting the `Conditions` and `Events` sections generated by the `describe`
+command, you determine the state of the DataVolume
+in relation to persistent volume claims (PVCs), and whether or
+not an operation is actively running or completed. You may also receive messages
+that offer specific details about the DataVolume’s status, and how
+it came to be in its current state.
+
+There are many different combinations of conditions. Each must be evaluated in its unique context.
+
+Examples of various combinations follow.
+
+* `Bound` – A successfully bound PVC displays in this example.
++
+Note that the `Type` is `Bound`, so the `Status` is `True`.
+If the PVC is not bound, the `Status` is `False`.
++
+When the PVC is bound, an event is generated stating that the PVC is bound.
+In this case, the `Reason` is `Bound` and `Status` is `True`.
+The `Message` indicates which PVC owns the DataVolume.
++
+`Message`, in the `Events` section, provides further details including how
+long the PVC has been bound (`Age`) and by what resource (`From`),
+in this case `datavolume-controller`.
++
+----
+Status:
+	Conditions:
+		Last Heart Beat Time:  2020-07-15T03:58:24Z
+		Last Transition Time:  2020-07-15T03:58:24Z
+		Message:               PVC win10-rootdisk Bound
+		Reason:                Bound
+		Status:                True
+		Type:                  Bound
+
+	Events:
+		Type     Reason     Age    From                   Message
+		----     ------     ----   ----                   -------
+		Normal   Bound      24s    datavolume-controller  PVC example-dv Bound
+----
+
+* `Running` – In this case, note that `Type` is `Running` and `Status` is `False`,
+indicating that an event has occurred that caused an attempted
+operation to fail, changing the Status from `True` to `False`.
++
+However, note that `Reason` is `Completed` and the `Message` field indicates
+`Import Complete`.
++
+In the `Events` section, the `Reason` and `Message` contain additional
+troubleshooting information about the failed operation. In this example,
+the `Message` displays an inability to connect due to a `404`, listed in the
+`Events` section’s first `Warning`.
++
+From this information, you conclude that an import operation was running,
+creating contention for other operations that are
+attempting to access the DataVolume.
```

Comment:
End with `:`.

Reply:
Also needs a `+` continuation for the code block that follows, since this is a bullet.

Reply:
Done.